docs/source/installing-brave.rst

```diff
@@ -4,7 +4,7 @@ Installing Brave
 Linux
 *****
 
-Brave supports 64-bit Linux distributions only.
+Brave supports 64-bit Linux distributions only (`amd64` / `x86_64`).
```

Comment:
This could possibly be a bit more descriptive with something like: "Brave supports 64-bit Linux distributions only on Intel/AMD architectures (`amd64` / `x86_64`)."